provider/schema/column.go

```diff
@@ -267,6 +267,6 @@ func (c Column) checkType(v interface{}) bool {
 			return true
 		}
 	}
-
-	return false
+	// everything can be a json
+	return c.Type == TypeJSON
```

Comment:
I disagree common types aren't allowed to be jsons, only structs, maps, arrays etc'